### ISSUE TITLE
Troubleshooting section

### ISSUE THREAD

**Issue by jschwarzwalder** [aws-samples/aws-modern-application-workshop#24]
It would be helpful if we added a section that will tell users how to view a created resource, update if they entered the wrong info the format time and to delete it so they can create again if it's not working.

I ended up deleting everything from the console and starting over because I didnt "create" a resource properly the first time and couldn't figure out how to override the values once the resource had been created.

**Comment by abaird986** [comment]
This is a good idea @jschwarzwalder, thank you for suggesting it!  I have put creating this type of section on our backlog for the tutorial. 

We have a couple more modules in the works right now, but a troubleshooting section for each module/section is a good one we should implement when time allows.

**Comment by smizles** [comment]
Is it possible this feature is still coming or already exists and I missed it somehow? I did the same thing as @jschwarzwalder where I couldn't quite untangle exactly where I had gone wrong in the tutorial. I was majorly frustrated when I saw there weren't any comments at the bottom of each module for users to discuss where they went wrong and what they did to correct it. At a minimum, a page/link on the actual tutorial for FAQ's would make quite a big difference IMHO.
